app/models/normalize_timdex.rb

```diff
@@ -38,17 +38,20 @@ def extract_title(record)
   end
 
   def extract_dates(dates)
-    # It is unlikely for a record to have more than one creation date, but just in case...
-    relevant_dates = dates.select { |date| date['kind'] == 'creation' }
+    # It is unlikely for a record to have more than one creation or publication date, but just in case...
+    relevant_dates = dates.select { |date| date['kind'] == 'creation' || date['kind'] == 'publication' }
 
-    # If the record *does* have more than one creation date, it's probably not worth determining which to display.
+    # If the record has no creation or publication date, stop here.
+    return if relevant_dates.empty?
+
+    # If the record *does* have more than one creation/pub date, it's probably not worth determining which to display.
     return if relevant_dates.count > 1
```

Comment:
I think now that we are checking both kinds of dates here we should remove this logic and just use the first value (or prioritize creation date or publication date if we know which is actually preferred here (which I don't)).